README.md

```diff
@@ -45,15 +44,18 @@ For a more detailed how to guide on setting them up [go to the Environment Varia
 
 6. [Make sure your database is running](#database_url) and setup the tables in the database with Drizzle by running:
 
+
 ```bash
 npm run db:push
 ```
+The full command can be seen in our [here](/package.json#L16) in our [package.json](/package.json) file
```

Comment:
Hey Edward. This markdown shows as:
The full command can be seen in our here in our package.json file

Maybe remove "in our here", so it appears as :
The full command can be seen in our package.json file

Reply:
Hi JP, yes, good spot! I'll change that before the final PR. Should be done by today.